src/ontology/fypo-odk.yaml

```diff
@@ -15,17 +15,33 @@ release_artefacts:
   - full
 primary_release: full
 import_group:
+  use_base_merging: TRUE
+  exclude_iri_patterns:
+    - <http://purl.obolibrary.org/obo/UBERON_*>
   products:
+    - id: go
+      make_base: TRUE
+      base_iris:
+        - http://purl.obolibrary.org/obo/GOCHE_
+        - http://purl.obolibrary.org/obo/GO_
+        - http://purl.obolibrary.org/obo/GOREL_
     - id: ro
+      use_base: TRUE
     - id: iao
+      use_base: TRUE
```

Comment:
Apparently IAO does not provide a base file (you can check that by trying to download `http://purl.obolibrary.org/obo/iao-base.owl`; you do get a file, but it’s not an ontology.

(It would probably be better if the PURL server could return a 404 in this kind of situation, but it does not. Not sure if it’s a bug, an oversight, or a behaviour that someone actually wanted. Anyway, that’s beyond the scope here.)